README.md

```diff
@@ -33,6 +33,9 @@ plugins: [
       // Optional: expediteBuild set this to either true or false
       expediteBuild: `boolean_value`,
 
+      // Optional: Specify true if you want to enable custom schema
+      isCustomSchemaEnable : `boolean_value`,
```

Comment:
Make functionality more explicit e.g.
"// Optional: Specify true if you want to generate custom schema"
Maybe add a link e.g.
"// Optional: Specify true if you want to generate [custom schema](https://www.gatsbyjs.org/docs/schema-generation/)"

Reply:
@stefanhar thanks for the your feedback. Also we are changing the key name from "isCustomSchemaEnable" to "enableSchemaGenenation".